src/navigation/_CollapsingNavigationMenu.jsx

```diff
@@ -102,13 +102,21 @@ class CollapsingNavigationMenu extends React.Component {
     this.renderMenuItems = this.renderMenuItems.bind(this);
     this.handleOnClick = this.handleOnClick.bind(this);
     this.handleKeyDown = this.handleKeyDown.bind(this);
+    this.selectedItem = React.createRef();
 
     this.state = {
       previousSelectedPath: selectedPath,
       openKeys: CollapsingNavigationMenu.openKeysToItem(menuItems[0], selectedPath),
     };
   }
 
+  componentDidMount() {
+    if (this.selectedItem && this.selectedItem.current) {
+      // window.scrollTo(0, this.selectedItem.current.offsetTop);
```

Comment:
Clean up?

Reply:
Thanks for catching that, I used Matt's commit suggestion: https://github.com/cerner/terra-dev-site/pull/217/commits/80b87d9fac768a94c053c5573696790815af890d